go.mod

```diff
@@ -29,21 +29,20 @@ require (
 	github.com/tchap/go-patricia/v2 v2.3.2
 	github.com/ulikunitz/xz v0.5.12
 	github.com/vbatts/tar-split v0.12.1
-	golang.org/x/exp v0.0.0-20241217172543-b2144cdd0a67
 	golang.org/x/sync v0.11.0
 	golang.org/x/sys v0.30.0
-	gotest.tools v2.2.0+incompatible
+	gotest.tools/v3 v3.5.2
```

Comment:
This clearly invites a question whether the project should be depending on both `github.com/stretchr/testify` and `gotest.tools/v3`.

Solving that is not a goal of this PR (which focuses on old/unmaintained/non-updated dependencies).